coops-ui/src/components/Common/SummaryOfficeAddresses.vue

```diff
@@ -0,0 +1,278 @@
+    'mailing-address': BaseAddress
+  }
+})
+export default class SummaryOfficeAddresses extends Mixins(CommonMixin, EntityFilterMixin) {
+  /**
+   * Registered Office address object passed in from the parent which is pulled from store.
+   */
+  @Prop({ default: null })
+  private registeredAddress: BaseAddressObjIF
+
+  /**
+   * Records Office address object passed in from the parent which is pulled from store.
+   */
+  @Prop({ default: null })
+  private recordsAddress: BaseAddressObjIF
+
+  // Entity Enum
+  private EntityTypes: {} = EntityTypes
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '../../assets/styles/theme.scss';
+  .v-btn {
+    margin: 0;
+    min-width: 6rem;
+    text-transform: none;
+  }
+
+  .reset-btn {
+    margin-top: 0.5rem;
+  }
+
+  .meta-container {
+    display: flex;
+    flex-flow: column nowrap;
+    position: relative;
+  }
+
+  .meta-container__inner {
+    margin-top: 1rem;
+  }
+
+  label:first-child {
+    font-weight: 700;
+    &__inner {
+      flex: 1 1 auto;
+    }
+  }
+
+  @media (min-width: 768px) {
+    .meta-container {
+      flex-flow: row nowrap;
+
+      label:first-child {
+        flex: 0 0 auto;
+        padding-right: 4rem;
+        width: 15.25rem;
+      }
+    }
+
+    .meta-container__inner {
+      margin-top: 0;
+    }
+  }
+
+  .address-list .form {
+    margin-top: 1rem;
+  }
+
+  @media (min-width: 768px) {
+    .address-list .form {
+      margin-top: 0rem
+    }
+  }
+
+  // Address Block Layout
+  .address-wrapper{
+    margin-top: .5rem;
+  }
+
+  .address-block__actions {
+    position: absolute;
+    top: 0;
+    right: 0;
+
+    .v-btn {
+      min-width: 4rem;
+    }
+
+    .v-btn + .v-btn {
+      margin-left: 0.5rem;
+    }
+  }
+
+  // Form Row Elements
+  .form__row + .form__row {
+    margin-top: 0.25rem;
+  }
+
+  .form__btns {
+    text-align: right;
+    display: flex;
+    justify-content: flex-end;
+    padding: 1rem;
+
+    .v-btn {
+      margin: 0;
+
+      + .v-btn {
+        margin-left: 0.5rem;
+      }
+    }
+  }
+
+  .form__row.three-column {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: stretch;
+    margin-right: -0.5rem;
+    margin-left: -0.5rem;
+  }
+
+  .inherit-checkbox {
+    margin-top: -3px;
+    margin-left: -3px;
+    padding: 0;
+  }
+  .records-inherit-checkbox {
+    margin-top: 0rem;
+    margin-left: 6rem;
+    margin-bottom: -1.5rem;
+    padding: 0;
+
+    .v-messages {
+      display: none!important;
+    }
+  }
+
+  // Registered Office Address Form Behavior
+  .show-address-form {
+
+    li:first-child {
+      padding-bottom: 0;
+    }
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  // Editing Address Form
+
+  .address-edit-header {
+    display: flex;
+    background-color: rgba(1,51,102,0.15);
+    padding: 1.25rem;
+
+    address-edit-title {
+      font-size: 16px;
+      font-weight: bold;
+      line-height: 22px;
+    }
+  }
+
+</style>
```

Comment:
I _think_ lint will ask you to add a blank line at the end of the file. If not, please add one anyway (for consistency).